frontend/integration-tests/tests/environment.scenario.ts

```diff
@@ -136,17 +148,37 @@ describe('Interacting with the environment variable editor', () => {
       const envPrefix = 'testcm';
       await environmentEditor(Actions.addFrom, resourceName, envPrefix);
       await environmentView.isLoaded();
-      await validateValueFrom(resourceName, envPrefix);
+      await validateValueFrom(resourceName, envPrefix, true);
     });
   });
 
-  xdescribe('CONSOLE-1504 - When a variable is added from a secret', () => {
+  describe('When a variable is deleted from a config map', () => {
```

Comment:
I'm ok adding this to our protractor test to get us past the flakes.  
But as we migrate to Cypress, we will not want tests that rely on the state of previous tests.

Reply:
Absolutely agree but did not want to tear this all out and refactor to just have to rewrite it in Cypres in a few months.